x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -432,6 +499,7 @@ export class EndpointDocGenerator {
         dataset: 'endpoint.metadata',
       },
       ...this.commonInfo,
+      ...createDataStreamField(metadataDataStream),
```

Comment:
❔ Just curious, why not just put `data_stream: metadataDataStream` there?

Reply:
Yeah I guess that's less verbose 👍